website/docs/guides/provider_reference.html.markdown

```diff
@@ -73,7 +73,7 @@ same configuration.
 ### Quick Reference
 
 * `credentials` - (Optional) Either the path to or the contents of a
-[service account key file] in JSON format. You can
+[service account key file] in JSON format. If you're using the file contents, be sure you replace the line breaks with an empty string. You can
```

Comment:
Do you know if this is true all the time,  or just when using TFC or TFE?

Reply:
All the time- you're setting an environment variable which can't have linebreaks.